doc/source/rest.j2

```diff
@@ -931,6 +931,22 @@ reporting values such as the number of new |measures| to process for each
 
 {{ scenarios['get-status']['doc'] }}
 
+Use-cases examples
+==================
+
+1. I want the top N instances for project_id "BD3A1E52-1C62-44CB-BF04-660BD88CD74D" that consume the most cpu last hour.
+
+To do so, I will use the :ref:`Dynamic aggregates API <aggregates>`. I will use
+|start|, |stop|, |granularity| and |groupby| parameters. The entire query will
+looke like:
```

Comment:
i imagine everything will probably go through aggregates endpoint so maybe drop this intro.